mirroring/handler.go

```diff
@@ -0,0 +1,230 @@
+package mirroring
+
+import (
+	"bufio"
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/go-openapi/strfmt"
+	trilliantypes "github.com/google/trillian/types"
+	"github.com/sigstore/rekor/pkg/generated/client"
+	"github.com/sigstore/rekor/pkg/generated/models"
+	"github.com/spf13/viper"
+)
+
+// consider adding a rekorClient field so that the same client struct is used for all operations, potentially saving some time
```

Comment:
is this a FIXME/TODO? can we label it as such? we could also turn it into a ticket.

Reply:
This was a TODO, but it's already done, so I deleted the message.